_sass/_hero.scss

```diff
@@ -0,0 +1,27 @@
+.hero {
+  align-items: center;
+  background: $ember-orange;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  justify-content: center;
+  width: 100vw;
```

Comment:
Are you using bourbon? 

Should these instead be `min-height`, and `min-width`?